src/components/ImageView/index.native.js

```diff
@@ -26,9 +29,7 @@ class ImageView extends React.Component {
 
     render() {
         // Default windowHeight accounts for the modal header height
-        const windowHeight = Dimensions.get('window').height - variables.contentHeaderHeight;
-        const windowWidth = Dimensions.get('window').width;
-
+        const windowHeight = this.props.windowHeight - variables.contentHeaderHeight;
```

Comment:
NAB maybe this local variable should be renamed `adjustedWindowHeight` to reduce any confusion ... I got confused for a sec and I think I might've written this code in the first place 😓

Reply:
This is sort of why I named it `windowHeight` instead of something even more generic like `height`. But it's reflecting the current dimensions of the window. I'm not sure we need the extra distinction between adjusted or not. If we need to know whether something changed we can compare `prevProps` and `this.props`.